code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -32,11 +32,13 @@
 /mob/living/simple_animal/hostile/zombie/proc/setup_visuals()
 	var/datum/job/job = SSjob.GetJob(zombiejob)
 
-	var/datum/outfit/outfit = new job.outfit
-	outfit.l_hand = null
-	outfit.r_hand = null
+	var/datum/outfit/outfit = job.outfits["Default"]?[SPECIES_HUMAN]
+	//outfit.l_hand = null
+	//outfit.r_hand = null
```

Comment:
if you're commenting out these lines, why not just delete them?

Reply:
i meant to delete them but i forgor